app/utils/jwt.js

```diff
@@ -7,7 +7,7 @@ export default Ember.Object.extend({
   payload: function(){
     var token = this.get('token');
 
-    if (!token) { return {}; }
+    if (!token || token === "null") { return {}; }
```

Comment:
Specifically this is a null value being written to localStorage. That is what stringifies.